lighthouse-core/report/scripts/lighthouse-report.js

```diff
@@ -47,9 +49,19 @@ LighthouseReport.prototype = {
     }
   },
 
+  toggleHelpText: function(e) {
+    if (e.target.classList.contains('report-section__item-help-toggle')) {
+      const el = e.currentTarget.parentNode.querySelector('.report-section__item-helptext');
```

Comment:
depending directly on the parentNode structure feels like smell, and potentially brittle code that could break with a trivial dom change.